app/index.html

```diff
@@ -16,5 +16,6 @@
   <script src="js/sumo_db.js"></script>
   <script src="js/notifications.js"></script>
   <script src="js/navigation.js"></script>
+  <script src="js/email_confirmation.js"></script>
```

Comment:
This should be above navigation.js because of how navigation.js inserts iframes. I'm sorry navigation.js is so fragile :(